src/main.rs

```diff
@@ -45,6 +45,10 @@
 //!     $ musl-gcc -static-pie -fPIC -o test test.c
 //!     $ target/debug/enarx-keepldr exec ./test
 //!     Hello World!
+//!
+//!     Or copy (or link) the files in external/*.service to ~/.config/systemd/user/
+//!     $ systemctl --user daemon-reload
+//!     $ systemctl --user start enarx-keep-nil@[Uuid].service
```

Comment:
I think this should not run in systemd `--user` mode, but system wide.

Reply:
I agree, but there's a balance between allowing multiple tests on a shared system and running as a particular user. I'm not sure what the correct approach is, tbh.

Reply:
Then, document: "for testing purposes […]"